### ISSUE TITLE
netdata: CONFIG: cannot load user config ‘/etc/netdata/stream.conf’. Will try stock config.

### ISSUE THREAD

**Issue by lichtamberg** [netdata/netdata#4403]
Since 1-2 weeks (cannot say exactly when), our logcheck reporting shows on multiple servers that netdata is not able to load the netdata /etc/netdata/stream.conf config file. When I look into /etc/netdata it shows me the entries below.

It looks like netdata has been updated and moved some config files into conf.d?
What am I supposed to do to fix this error? Unfortunately I cannot find any information in google  or the README, so I'm asking here.

It probably would be good to add some text when generating the .installer-cleanup-of-stock-configs-done file, so it's clear how to proceed here.

This line moved the configuration files as it seems: https://github.com/netdata/netdata/blob/master/makeself/install-or-update.sh#L160

```shell
user@host:/etc/netdata$ ll
total 172
drwxr-xr-x   7 root netdata  4096 Oct 16 06:33 ./
drwxr-xr-x 112 root root     4096 Jun  5 11:48 ../
lrwxrwxrwx   1 root netdata    23 Sep 28 06:47 000.-.USE.THE.orig.LINK.TO.COPY.AND.EDIT.STOCK.CONFIG.FILES -> /usr/lib/netdata/conf.d/
-rw-r-----   1 root netdata  9449 Apr  5  2018 apps_groups.conf
-rw-r-----   1 root netdata  9449 Apr  5  2018 apps_groups.conf.old
drwxr-xr-x   2 root netdata  4096 Sep 28 06:47 charts.d/
-rwxr-xr-x   1 root netdata  2284 Oct 16 06:33 edit-config*
-rw-r-----   1 root netdata 25751 Apr  5  2018 health_alarm_notify.conf
-rw-r-----   1 root netdata 25737 Apr  5  2018 health_alarm_notify.conf.bak
-rw-r-----   1 root netdata 25751 Apr  5  2018 health_alarm_notify.conf.old
drwxr-xr-x   2 root netdata 12288 Sep 28 06:47 health.d/
-rw-r-----   1 root netdata     0 Sep 28 06:47 .installer-cleanup-of-stock-configs-done
-rw-r-----   1 root netdata   218 Apr  5  2018 netdata.conf
-rw-r-----   1 root netdata   218 Apr  5  2018 netdata.conf.old
drwxr-xr-x   2 root netdata  4096 Oct 14 06:39 node.d/
lrwxrwxrwx   1 root root       23 Oct 16 06:33 orig -> /usr/lib/netdata/conf.d/
drwxr-xr-x   2 root netdata 12288 Sep 28 06:47 python.d/
-rw-r-----   1 root netdata  1537 May 26 06:52 python.d.conf
-rw-r-----   1 root netdata  1537 May 26 06:52 python.d.conf.old
drwxr-xr-x   2 root netdata  4096 Sep 28 06:47 statsd.d/
user@host:/etc/netdata$ ll orig/
total 104
drwxr-xr-x 7 root netdata  4096 Oct 16 06:33 ./
drwxrwxr-x 3 root root     4096 Sep 28 06:47 ../
-rw-r----- 1 root netdata  9606 Oct 16 06:33 apps_groups.conf
drwxr-xr-x 2 root netdata  4096 Oct 16 06:33 charts.d/
-rw-r----- 1 root netdata  1829 Oct 16 06:33 charts.d.conf
-rw-r----- 1 root netdata  1233 Oct 16 06:33 fping.conf
-rw-r----- 1 root netdata 35872 Oct 16 06:33 health_alarm_notify.conf
drwxr-xr-x 2 root netdata  4096 Oct 16 06:33 health.d/
-rw-r----- 1 root netdata    57 Oct 16 06:33 health_email_recipients.conf
drwxr-xr-x 2 root netdata  4096 Oct  1 06:32 node.d/
-rw-r----- 1 root netdata  1310 Oct 16 06:33 node.d.conf
drwxr-xr-x 2 root netdata  4096 Oct 16 06:33 python.d/
-rw-r----- 1 root netdata  1830 Oct 16 06:33 python.d.conf
drwxr-xr-x 2 root netdata  4096 Oct 16 06:33 statsd.d/
-rw-r----- 1 root netdata  7564 Oct 16 06:33 stream.conf
```

**Comment by Ferroin** [comment]
@ktsaou We should probably either hide this error, or make a point to document somewhere that it's non-fatal as long as you're not using streaming.

**Comment by ktsaou** [comment]
Well, this is `INFO` not `ERROR`: https://github.com/netdata/netdata/blob/8fbf817ef83b3524b15f908251909d9d6feb5532/daemon/main.c#L652

In addition streaming is enabled with this file. If we hide it, then people will not be able to find out why they cannot enable it.

So, I don't think we can make it better.
I am open to ideas though...

**Comment by Ferroin** [comment]
It could be made better by not printing it to the terminal...

Most of the issue here is that it gets spit out on whatever the controlling terminal.  Previously, startup of netdata was silent if there wasn't an error, now it isn't, so most people will assume something is wrong.

Is there some particular reason we have to try to load the stream configuration _before_ we have a working log file?  Because it makes a whole lot more sense from my point of view to get the core configuration working and _then_ try to load secondary stuff like this.

**Comment by ktsaou** [comment]
Fixed with PR #4422
